### PR TITLE
A Tiny JS World

### PR DESCRIPTION
**Pull request by DeamonFire**
### A Tiny JS World

[Demo](https://deamonfire.github.io/a-tiny-JS-world/)|
[Code base](https://github.com/DeamonFire/a-tiny-JS-world/blob/master/index.js)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by DeamonFire** [comment]
Hi, I've checked the code for common errors as well as meeting the basic requirements of the initial task.

Please review =)

File: submissions/DeamonFire/A-Tiny-JS-World/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+   say: 'Death to enemies!'
+};
+
+const catWoman = {
+   species: 'catHuman',
+   name: 'Ania',
+   gender: 'female',
+   hands: 2,
+   legs: 2,
+   say: cat.say
```
Comment (OleksiyRudenko) [inline submissions/DeamonFire/A-Tiny-JS-World/index.js:52]: Will cat-woman's saying change automagically when cat's does?

File: submissions/DeamonFire/A-Tiny-JS-World/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+};
+
+const habitansWorld = [dog, cat, woman, man, catWoman];
+const props = ['species', 'name', 'gender', 'hands', 'legs', 'say'];
+
+// ======== OUTPUT ========
+const residents = habitansWorld
```
Comment (OleksiyRudenko) [inline submissions/DeamonFire/A-Tiny-JS-World/index.js:59]: Is it residents per se that are stored in this variable? Looks like residents is synonym to "habitans".
Think of the naming that describes what is actually stored in a variable.

File: submissions/DeamonFire/A-Tiny-JS-World/index.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+const catWoman = {
+   species: 'catHuman',
+   name: 'Ania',
+   gender: 'female',
+   hands: 2,
+   legs: 2,
+   say: cat.say
+};
+
+const habitansWorld = [dog, cat, woman, man, catWoman];
```
Comment (OleksiyRudenko) [inline submissions/DeamonFire/A-Tiny-JS-World/index.js:55]: I often wonder where the wrong word "habitans" comes from in many works. Looks like it spreads as a pandemic across submissions.

**Review by OleksiyRudenko — APPROVED** [review]
@DeamonFire well done.
Check the comments below and put down a couple of insights for yourself.

**Pull request merged**